1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0, 6);
+names[1] = names[1].substring(0, 7);
+names[2] = names[2].substring(0, 4);
+names[3] = names[3].substring(0, 4);
+names[4] = names[4].substring(0, 5);
```

Comment:
Yes, this works well. You could also find the index of the first space and use that variable to find the end of the substring - although that could cause problems with people with more than 2 names, so I think what you've done is fine.